modules/apple_support/0.11.0/presubmit.yml

```diff
@@ -0,0 +1,9 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
```

Comment:
I'd be surprised if things passed on windows

Reply:
Yeah, the tests were failing on everything but macOS. So we run tests only on macOS but we can make sure the exposed targets at least load correctly: https://github.com/bazelbuild/bazel-central-registry/pull/63#discussion_r786890287

Reply:
This should probably also be added back to the test setup of the apple support, because we don't want to detect the failure only when the release is done and we're already checking it into the BCR.